test/e2e/latency.go

```diff
@@ -0,0 +1,271 @@
+/*
```

Comment:
Can you please refactor it somehow (maybe by putting the main logic to the new "perf_util.go" file) so that we can just measure the startup latency in Density test too?
It's quite important that we don't add a new test to our scalability suite now (as long as deleting namespace after them takes 30m +).

Ideally, how I would see it, is that we have some helper function:
"testPodStartupLatency(client, /*maybe sth else*/)"
that we could call from both Density tests and possibly from the new test you created here.